### DISCUSSION TITLE
Unable to login beacuse confirmation email never recieved

### DISCUSSION THREAD

**Discussion by Sypher1k** [chatwoot/chatwoot#1625] [Q&A]
I have installed Chatwoot on AWS EC2 Ubuntu and running Chatwoot dev on docker.

I'm able to create a new user account and add a password but the confirmation email never gets delivered therefore I'm unable to sign back in without confirming the email address. 

** _Below is the work log after creating a new account_**

webpack_1   | ��� ���wdm���: Compiled successfully.
rails_1     | Started GET "/" for 61.245.171.185 at 2021-01-07 06:53:37 +0000
rails_1     | Processing by DashboardController#index as HTML
rails_1     |   Rendering dashboard/index.html.erb within layouts/vueapp
rails_1     |   Rendered dashboard/index.html.erb within layouts/vueapp (Duration: 0.0ms | Allocations: 4)
rails_1     | Completed 200 OK in 11ms (Views: 9.8ms | ActiveRecord: 0.0ms | Allocations: 2296)
rails_1     |
rails_1     |
rails_1     | Started POST "/api/v1/accounts.json" for 61.245.171.185 at 2021-01-07 06:53:57 +0000
rails_1     | Processing by Api::V1::AccountsController#create as JSON
rails_1     |   Parameters: {"account_name"=>"Sypher", "user_full_name"=>"Bha**", "email"=>"bha****@hotmail.com", "account"=>{}}
rails_1     | Enqueued EventDispatcherJob (Job ID: 685e0d71-962b-4107-9cec-540a89f6aa57) to Sidekiq(events) with arguments: "account.created", Thu, 07 Jan 2021 06:53:57 UTC +00:00, {:account=>#<GlobalID:0x000055b4b8aad198 @uri=#<URI::GID gid://chatwoot/Account/2>>}
rails_1     | Enqueued ActionMailer::MailDeliveryJob (Job ID: 90c809a9-3629-422e-976b-1101cbf005e2) to Sidekiq(mailers) with arguments: "Devise::Mailer", "confirmation_instructions", "deliver_now", {:args=>[#<GlobalID:0x000055b4bc034af0 @uri=#<URI::GID gid://chatwoot/User/2>>, "6_pboS1CgFjQhUsrA5fW", {:client_config=>"default", :redirect_url=>nil}]}
rails_1     | Enqueued EventDispatcherJob (Job ID: 83de6f9c-7e7a-40e1-a673-865e581e2cc7) to Sidekiq(events) with arguments: "agent.added", Thu, 07 Jan 2021 06:53:58 UTC +00:00, {:account=>#<GlobalID:0x000055b4bbf06548 @uri=#<URI::GID gid://chatwoot/Account/2>>}
rails_1     |   Rendering api/v1/accounts/create.json.jbuilder
rails_1     |   Rendered api/v1/accounts/create.json.jbuilder (Duration: 164.9ms | Allocations: 26294)
rails_1     | Completed 200 OK in 1002ms (Views: 188.3ms | ActiveRecord: 87.9ms | Allocations: 153396)
rails_1     |
rails_1     |
rails_1     | Started GET "/" for 61.245.171.185 at 2021-01-07 06:53:59 +0000
rails_1     | Processing by DashboardController#index as HTML
rails_1     |   Rendering dashboard/index.html.erb within layouts/vueapp
rails_1     |   Rendered dashboard/index.html.erb within layouts/vueapp (Duration: 0.3ms | Allocations: 72)
rails_1     | Completed 200 OK in 14ms (Views: 11.7ms | ActiveRecord: 0.0ms | Allocations: 9908)
rails_1     |
rails_1     |
rails_1     | Started GET "/api/v1/accounts/2/labels" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Processing by Api::V1::Accounts::LabelsController#index as JSON
rails_1     |   Parameters: {"account_id"=>"2"}
rails_1     |   Rendering api/v1/accounts/labels/index.json.jbuilder
rails_1     |   Rendered api/v1/accounts/labels/index.json.jbuilder (Duration: 3.6ms | Allocations: 1834)
rails_1     | Completed 200 OK in 102ms (Views: 9.5ms | ActiveRecord: 3.4ms | Allocations: 15644)
rails_1     |
rails_1     |
rails_1     | Started GET "/cable" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Started GET "/cable/" [WebSocket] for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: Upgrade, HTTP_UPGRADE: websocket)
rails_1     | Started GET "/api/v1/accounts/2/inboxes" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Processing by Api::V1::Accounts::InboxesController#index as JSON
rails_1     | Started GET "/api/v1/accounts/2/conversations?status=open&assignee_type=me&page=1" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     |   Parameters: {"account_id"=>"2"}
rails_1     | Processing by Api::V1::Accounts::ConversationsController#index as JSON
rails_1     |   Parameters: {"status"=>"open", "assignee_type"=>"me", "page"=>"1", "account_id"=>"2"}
rails_1     |   Rendering api/v1/accounts/inboxes/index.json.jbuilder
rails_1     |   Rendered api/v1/accounts/inboxes/index.json.jbuilder (Duration: 4.5ms | Allocations: 2942)
rails_1     | Completed 200 OK in 74ms (Views: 10.6ms | ActiveRecord: 21.2ms | Allocations: 38849)
rails_1     |
rails_1     |
rails_1     | Creating scope :open. Overwriting existing method Conversation.open.
rails_1     |   Rendering api/v1/accounts/conversations/index.json.jbuilder
rails_1     |   Rendered api/v1/accounts/conversations/index.json.jbuilder (Duration: 3.5ms | Allocations: 1866)
rails_1     | Completed 200 OK in 68ms (Views: 6.1ms | ActiveRecord: 11.0ms | Allocations: 35875)
rails_1     |
rails_1     |
rails_1     | Started GET "/api/v1/accounts/2/agents" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Processing by Api::V1::Accounts::AgentsController#index as JSON
rails_1     |   Parameters: {"account_id"=>"2"}
rails_1     |   Rendering api/v1/accounts/agents/index.json.jbuilder
rails_1     |   Rendered api/v1/models/_agent.json.jbuilder (Duration: 15.4ms | Allocations: 6639)
rails_1     |   Rendered api/v1/accounts/agents/index.json.jbuilder (Duration: 23.8ms | Allocations: 10514)
rails_1     | Completed 200 OK in 42ms (Views: 24.8ms | ActiveRecord: 3.2ms | Allocations: 19309)
rails_1     |
rails_1     |
rails_1     | Started GET "/auth/validate_token" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Started GET "/api/v1/accounts/2/" for 61.245.171.185 at 2021-01-07 06:54:00 +0000
rails_1     | Processing by Api::V1::AccountsController#show as JSON
rails_1     |   Parameters: {"id"=>"2"}
rails_1     |   Rendering api/v1/accounts/show.json.jbuilder
rails_1     |   Rendered api/v1/accounts/show.json.jbuilder (Duration: 1.5ms | Allocations: 969)
rails_1     | Processing by DeviseOverrides::TokenValidationsController#validate_token as HTML
rails_1     | Completed 200 OK in 38ms (Views: 9.5ms | ActiveRecord: 9.6ms | Allocations: 14605)
rails_1     |
rails_1     |
rails_1     |   Rendering devise/token.json.jbuilder
rails_1     |   Rendered api/v1/models/_user.json.jbuilder (Duration: 19.4ms | Allocations: 15320)
rails_1     |   Rendered devise/_auth.json.jbuilder (Duration: 24.0ms | Allocations: 15613)
rails_1     |   Rendered devise/token.json.jbuilder (Duration: 30.7ms | Allocations: 18688)
rails_1     | Completed 200 OK in 42ms (Views: 34.7ms | ActiveRecord: 3.3ms | Allocations: 25595)
rails_1     |
rails_1     |
rails_1     | RoomChannel is streaming from ATqk8yAWjYa5VhAGwV7vC68x
rails_1     | RoomChannel is transmitting the subscription confirmation
mailhog_1   | [APIv1] KEEPALIVE /api/v1/events

--------------------------------------------------------------------------------------------------------------------------------
My Env SMTP setting are:

//Mail outgoing
MAILER_SENDER_EMAIL=jake.xxxxxx@gmail.com
SMTP_PORT=465
SMTP_DOMAIN=smtp.gmail.com
//if you are running docker-compose, set SMTP_ADDRESS value as "mailhog",
// else set the value as "localhost"
SMTP_ADDRESS=mailhog
SMTP_USERNAME=jake.xxxxxx@gmail.com
SMTP_PASSWORD=xxxxxxxxx
SMTP_AUTHENTICATION=login
SMTP_ENABLE_STARTTLS_AUTO=true

__________________________________________________________________________________________________________________________
Get this back from signin request:

![image](https://user-images.githubusercontent.com/20921224/103863509-0070ec00-50e7-11eb-8142-f19bc86f2847.png)

Thank you in advance :)

**Comment by pranavrajs** [comment]
SMTP_ADDRESS should be `smtp.gmail.com` instead of `mailhog`. You don't need to set SMTP_DOMAIN.

**Comment by Sypher1k** [comment]
@pranavrajs Thanks for the quick reply
I changed SMTP_ADDRESS in .env and didn't work.

### Work log after creating new account

Processing by Api::V1::AccountsController#create as JSON
rails_1     |   Parameters: {"account_name"=>"Bhanu", "user_full_name"=>"S Rachchige", "email"=>"jake.xxxx@gmail.com", "account"=>{}}
rails_1     | Enqueued EventDispatcherJob (Job ID: 0555ae60-49d8-427a-8bda-1ab2dfebca90) to Sidekiq(events) with arguments: "account.created", Thu, 07 Jan 2021 08:40:19 UTC +00:00, {:account=>#<GlobalID:0x000055be22dc5948 @uri=#<URI::GID gid://chatwoot/Account/3>>}
rails_1     | Enqueued ActionMailer::MailDeliveryJob (Job ID: 1045c452-4c13-490c-9ac3-99a79eedc47d) to Sidekiq(mailers) with arguments: "Devise::Mailer", "confirmation_instructions", "deliver_now", {:args=>[#<GlobalID:0x000055be226571c0 @uri=#<URI::GID gid://chatwoot/User/3>>, "1Z3NfNJU62Wwaje1K_2b", {:client_config=>"default", :redirect_url=>nil}]}
rails_1     | Enqueued EventDispatcherJob (Job ID: 08b2c201-7244-4ee5-acae-3501888600a8) to Sidekiq(events) with arguments: "agent.added", Thu, 07 Jan 2021 08:40:19 UTC +00:00, {:account=>#<GlobalID:0x000055be22323350 @uri=#<URI::GID gid://chatwoot/Account/3>>}
rails_1     |   Rendering api/v1/accounts/create.json.jbuilder
rails_1     |   Rendered api/v1/accounts/create.json.jbuilder (Duration: 15.1ms | Allocations: 10815)
rails_1     | Completed 200 OK in 342ms (Views: 16.4ms | ActiveRecord: 24.3ms | Allocations: 49595)
rails_1     |
rails_1     |
rails_1     | Started GET "/" for 124.43.13.12 at 2021-01-07 08:40:19 +0000
rails_1     | Processing by DashboardController#index as HTML
rails_1     |   Rendering dashboard/index.html.erb within layouts/vueapp
rails_1     |   Rendered dashboard/index.html.erb within layouts/vueapp (Duration: 0.0ms | Allocations: 4)
rails_1     | Completed 200 OK in 6ms (Views: 4.6ms | ActiveRecord: 0.0ms | Allocations: 2679)
rails_1     |
rails_1     |
mailhog_1   | [APIv1] KEEPALIVE /api/v1/events
rails_1     | Started GET "/api/v1/accounts/3/labels" for 124.43.13.12 at 2021-01-07 08:40:21 +0000
rails_1     | Processing by Api::V1::Accounts::LabelsController#index as JSON
rails_1     |   Parameters: {"account_id"=>"3"}
rails_1     |   Rendering api/v1/accounts/labels/index.json.jbuilder
rails_1     |   Rendered api/v1/accounts/labels/index.json.jbuilder (Duration: 3.0ms | Allocations: 1604)
rails_1     | Completed 200 OK in 85ms (Views: 4.7ms | ActiveRecord: 2.0ms | Allocations: 7949)
rails_1     |
rails_1     |
rails_1     | Started GET "/cable" for 124.43.13.12 at 2021-01-07 08:40:21 +0000
rails_1     | Started GET "/cable/" [WebSocket] for 124.43.13.12 at 2021-01-07 08:40:21 +0000
rails_1     | Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: Upgrade, HTTP_UPGRADE: websocket)
rails_1     | Started GET "/api/v1/accounts/3/conversations?status=open&assignee_type=me&page=1" for 124.43.13.12 at 2021-01-07 08:40:21 +0000
rails_1     | Processing by Api::V1::Accounts::ConversationsController#index as JSON
rails_1     | Started GET "/api/v1/accounts/3/inboxes" for 124.43.13.12 at 2021-01-07 08:40:21 +0000
rails_1     |   Parameters: {"status"=>"open", "assignee_type"=>"me", "page"=>"1", "account_id"=>"3"}
rails_1     | Processing by Api::V1::Accounts::InboxesController#index as JSON
rails_1     |   Parameters: {"account_id"=>"3"}
rails_1     |   Rendering api/v1/accounts/inboxes/index.json.jbuilder
rails_1     |   Rendered api/v1/accounts/inboxes/index.json.jbuilder (Duration: 3.8ms | Allocations: 2642)
rails_1     | Completed 200 OK in 29ms (Views: 7.1ms | ActiveRecord: 6.0ms | Allocations: 18155)

----------------------------------------------------------------------------------------------------------------------------------------------------

###  docker .env file

SECRET_KEY_BASE=replace_with_lengthy_secure_hex

FRONTEND_URL=http://*.**.207.105:3000

ASSET_CDN_HOST=

FORCE_SSL=false

ENABLE_ACCOUNT_SIGNUP=true

REDIS_URL=redis://redis:6379
REDIS_PASSWORD=
REDIS_SENTINELS=

REDIS_SENTINEL_MASTER_NAME=

POSTGRES_HOST=postgres
POSTGRES_USERNAME=postgres
POSTGRES_PASSWORD=password
RAILS_ENV=development
RAILS_MAX_THREADS=5

MAILER_SENDER_EMAIL=jake.****@gmail.com
SMTP_PORT=465
SMTP_SERVER=
SMTP_ADDRESS=smtp.gmail.com
SMTP_USERNAME=jake.*****@gmail.com
SMTP_PASSWORD=********
SMTP_AUTHENTICATION=login
SMTP_ENABLE_STARTTLS_AUTO=true

MAILER_INBOUND_EMAIL_DOMAIN=
RAILS_INBOUND_EMAIL_SERVICE=
RAILS_INBOUND_EMAIL_PASSWORD=
MAILGUN_INGRESS_SIGNING_KEY=
MANDRILL_INGRESS_API_KEY=

ACTIVE_STORAGE_SERVICE=local

S3_BUCKET_NAME=
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_REGION=

SENTRY_DSN=

RAILS_LOG_TO_STDOUT=true
LOG_LEVEL=info
LOG_SIZE=500
FB_VERIFY_TOKEN=
FB_APP_SECRET=
FB_APP_ID=

TWITTER_APP_ID=
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_ENVIRONMENT=

#slack integration
SLACK_CLIENT_ID=
SLACK_CLIENT_SECRET=

IOS_APP_ID=6C953F3RX2.com.chatwoot.app

USE_INBOX_AVATAR_FOR_BOT=true

**Comment by helpobras** [comment]
Friend, how did you solve this? I'm not getting it. is there any way to confirm manually?


MAILER_SENDER_EMAIL=suporte@help-obras.com.br
SMTP_DOMAIN=help-obras.com.br
SMTP_ADDRESS=help-obras.com.br
SMTP_PORT=465
SMTP_USERNAME=suporte@help-obras.com.br
SMTP_PASSWORD=xxxxxxxxxx
SMTP_AUTHENTICATION=login
SMTP_ENABLE_STARTTLS_AUTO=true
SMTP_OPENSSL_VERIFY_MODE=peer

**Reply by sojan-official** [reply]
https://github.com/chatwoot/chatwoot/issues/2513#issuecomment-868759346

**Reply by helpobras** [reply]
Perfect, thanks.

**Comment by helpobras** [comment]
continuing the part of the email settings:
this time I created an account on gmail...I made the releases and on rails I have the following LOG.
[(<html>
<body>


Net::SMTPAuthenticationError: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbu
--


<br class="Apple-interchange-newline">
</body>
</html>)]

**Reply by helpobras** [reply]
it was gmail blocking.. i managed to unblock
